app/components/inventory/rank-select.js

```diff
@@ -0,0 +1,16 @@
+import Ember from 'ember';
+import SelectValues from 'hospitalrun/utils/select-values';
+
+const c = Ember.computed;
+
+export default Ember.Component.extend({
+  label: 'Rank',
+  rankOptions: ['A', 'B', 'C'],
```

Comment:
actual options (Array of Objects) is defined via CP in [here](https://github.com/HospitalRun/hospitalrun-frontend/pull/225/files#diff-0efd4da99b10af26a375b4c06eabe7baR13). I can define setter for this in `init()` if you like to.